crypto/evp/evp_fetch.c

```diff
@@ -242,9 +277,19 @@ inner_evp_generic_fetch(OPENSSL_CTX *libctx, int operation_id,
      * evp_method_id returns 0 if we have too many operations (more than
      * about 2^8) or too many names (more than about 2^24).  In that case,
      * we can't create any new method.
+     * For all intents and purposes, this is an internal error.
      */
-    if (name_id != 0 && (meth_id = evp_method_id(name_id, operation_id)) == 0)
+    if (name_id != 0 && (meth_id = evp_method_id(name_id, operation_id)) == 0) {
```

Comment:
The `name_id` can come from the user which shouldn't be an internal error should it?

Reply:
Actually, it is, because all the fetch by number functions are internal.  So, the conditions for failure is that we have let the `OSSL_OP_` numbers run too high (i.e. past 255), or that there are sooo many algorithms that their numbers (name_id) go past 2^24-1.

Reply:
I look forward to the day where name_id breaks the 2^24-1 barrier :wink: